request.d.ts compile error for browsers

When I try to build my project with `tsc && vite build` it fails due to the following exception:

```
node_modules/ky/distribution/types/request.d.ts:2:113 - error TS2694: Namespace 'global.NodeJS' has no exported member 'ArrayBufferView'.

2 type UndiciBodyInit = ArrayBuffer | AsyncIterable<Uint8Array> | Blob | FormData | Iterable<Uint8Array> | NodeJS.ArrayBufferView | URLSearchParams | null | string;
                                                                                                                  ~~~~~~~~~~~~~~~


Found 1 error in node_modules/ky/distribution/types/request.d.ts:2
```

--- a/source/types/request.ts
+++ b/source/types/request.ts
@@ -16,7 +16,7 @@ type UndiciBodyInit =
 	| Blob
 	| FormData
 	| Iterable<Uint8Array>
-	| NodeJS.ArrayBufferView
+	| ArrayBufferView
 	| URLSearchParams
 	// eslint-disable-next-line @typescript-eslint/ban-types
 	| null

